ftw/news/tests/test_news_listing.py

```diff
@@ -0,0 +1,144 @@
+        browser.login(username=self.member.getProperty('id'),
+                      password=DEFAULT_PASSWORD).open(self.news_folder)
+        self.assertTrue(self._is_author_visible(browser),
+                        'Authenticated member should see author if '
+                        'allowAnonymousViewAbout is False.')
+
+    @browsing
+    def test_anonymous_sees_author_when_aava_enabled(self, browser):
+        set_allow_anonymous_view_about(self.news_folder, True)
+        browser.open(self.news_folder)
+        self.assertTrue(self._is_author_visible(browser),
+                        'Anonymous user should see author if '
+                        'allowAnonymousViewAbout is True.')
+
+    @browsing
+    def test_anonymous_cannot_see_author_when_aava_disabled(self, browser):
+        set_allow_anonymous_view_about(self.news_folder, False)
+        browser.open(self.news_folder)
+        self.assertTrue(self._is_author_visible(browser),
+                        'Anonymous user should not see author if '
+                        'allowAnonymousViewAbout is False.')
+
+    @browsing
+    def test_inactive_news_is_visible_for_contributor(self, browser):
+        browser.login(username=self.contributor.getProperty('id'),
+                      password=DEFAULT_PASSWORD).open(self.news_folder)
+        self.assertEqual(2, len(browser.css('div.tileItem')))
+
+    @browsing
+    def test_inactive_news_is_not_visible_for_regular_users(self, browser):
+        browser.login(username=self.member.getProperty('id'),
+                      password=DEFAULT_PASSWORD).open(self.news_folder)
+        self.assertEqual(1, len(browser.css('div.tileItem')))
+
+    @browsing
+    def test_news_listing_inside_contentpage(self, browser):
+        page = create(Builder('sl content page').titled(u'Content page'))
+        news_folder = create(Builder(u'news folder')
+                             .titled(u'A News Folder')
+                             .within(page))
+
+        browser.login().visit(page, view='@@news_listing')
+        self.assertEquals('Content page - News', plone.first_heading())
+
+        browser.visit(news_folder)
+        self.assertEquals(u'A News Folder', plone.first_heading())
+
+    @browsing
+    def test_news_listing_title(self, browser):
+        """
+        This test makes sure that the news listing view renders the
+        title of the portlet, not the generic title.
+        """
+        browser.login().visit(self.news_folder, view='news_listing')
+        self.assertEquals(
+            ['A News Folder'],
+            browser.css('h1.documentFirstHeading').text
+        )
+
+    @browsing
+    def test_news_listing_no_description(self, browser):
+        """
+        This test makes sure that the news listing view does not
+        render the generic description.
+        """
+        browser.login().visit(self.news_folder, view='news_listing')
+        self.assertIsNone(
```

Comment:
is there even a description which is not renderend?
it would be nice if this would be verified here in order to make sure that the test actually tests what he tells.
e.g.
```python
self.assertTrue(self.news_folder.Description())
```